extensions/resteasy-common/deployment/src/main/java/io/quarkus/resteasy/common/deployment/ResteasyCommonProcessor.java

```diff
@@ -195,9 +194,15 @@ private boolean restJsonSupportNeeded(CombinedIndexBuildItem indexBuildItem, Dot
                 continue;
             }
 
-            final List<String> mediaTypes = Arrays.asList(annotationValue.asStringArray());
-            return mediaTypes.contains(MediaType.APPLICATION_JSON)
-                    || mediaTypes.contains(MediaType.APPLICATION_JSON_PATCH_JSON);
+            List<String> mediaTypes = null;
```

Comment:
Ah. I totally missed the second one was not a `else`. Probably better to initialize that one to `Collections.emptyList()` then.

That way, you don't have to special case the null.

Sorry about that.

Reply:
sure